docs/decisions/infra/0008-consolidate-infra-config-from-tfvars-files-into-config-module.md

```diff
@@ -0,0 +1,29 @@
+# Consolidate infra configuration from .tfvars files into config module
+
+* Status: proposed
+* Deciders: @lorenyu @kyeah @acouch
```

Comment:
Could I get added to this list or is it best to keep it to the people who came up with the original idea?

Reply:
Definitely will add you!

Reply:
Thanks for the reminder!